mama/c_cpp/src/gunittest/c/MainUnitTestC.cpp

```diff
@@ -60,16 +84,36 @@ static void parseCommandLine (int argc, char** argv)
 
     for (i = 1; i < argc;)
     {
-		if (strcmp ("-m", argv[i]) == 0)
-		{
-			gMiddleware = argv[i+1];
-			i += 2;
-		}
-		else if (strcmp ("-p", argv[i]) == 0)
-		{
-			gPayload = argv[i+1];
-			i += 2;
-		}
+        if (strcmp ("-m", argv[i]) == 0)
+        {
+            gMiddleware = argv[i+1];
+            i += 2;
+        }
+        else if (strcmp ("-p", argv[i]) == 0)
+        {
+            gPayload = argv[i+1];
+            i += 2;
+        }
+        else if (strcmp ("-tport", argv[i]) == 0)
```

Comment:
The issue here is that the defaults don't run for some environments. If a tester does not have wmw installed then the unit tests will fail. I run with Solace and Tick42/RMDS for unit tests, I need to be able to specify the parms. All of the unit tests I run are scripted so adding the cmd line parms is easier.